app/components/policy-card.js

```diff
@@ -16,8 +32,14 @@ export default Component.extend({
   }),
   didRender() {
     this._super(...arguments);
-    if (this.get('methodAJournal') && !this.get('removeNIHDeposit')) {
```

Comment:
Is the `removeNIHDeposit` condition no longer applicable after refactoring?

Reply:
I did a light sweep of removeNIHDeposit property because it was appearing in multiple places and there were two properties doing very similar things - it's one of the few I was forced to refactor a bit and ultimately eliminate. The removal happens on `app/components/workflow-repositories.js#53`.  The logic does the same thing. If pmcPublisherDeposit is true, it will remove the NIH repository. If the journal is methodA, pmcPublisherDeposit should always be true.